softwareComponents/simplesimClientLib/src/changecolor.cpp

```diff
@@ -0,0 +1,97 @@
+#include "changecolor.hpp"
+#include "ui_changecolor.h"
+
+#include <iostream>
+
+using rofi::simplesim::ChangeColor;
+
+ChangeColor::ChangeColor( QWidget* parent, size_t size ) :
+    ui( new Ui::ChangeColor ),
+    parent( parent ),
+    to_color( size, false )
+{
+    //QWidget( parent );
```

Comment:
What's this supposed to mean? (I know, it is commented-out, but still)

Reply:
Relic that doesn't have to be there, deleted